components/widgets/inputs/users_emails_input.jsx

```diff
@@ -258,32 +264,54 @@ export default class UsersEmailsInput extends React.PureComponent {
-                placeholder={this.props.placeholder}
-                components={this.components}
-                getOptionValue={this.getOptionValue}
-                formatOptionLabel={this.formatOptionLabel}
-                defaultOptions={false}
-                defaultMenuIsOpen={false}
-                openMenuOnClick={false}
-                loadingMessage={this.loadingMessage}
-                onInputChange={this.handleInputChange}
-                inputValue={this.props.inputValue}
-                openMenuOnFocus={true}
-                onFocus={this.onFocus}
-                tabSelectsValue={true}
-                value={values}
-                aria-label={this.props.ariaLabel}
-            />
+            <div>
+                <AsyncSelect
+                    ref={this.selectRef}
+                    styles={this.customStyles}
+                    onChange={this.onChange}
+                    loadOptions={this.optionsLoader}
+                    isValidNewOption={this.showAddEmail}
+                    isMulti={true}
+                    isClearable={false}
+                    className={classNames(
+                        'UsersEmailsInput',
+                        this.props.showError ? 'error' : '',
+                        {empty: this.props.inputValue === ''},
+                    )}
+                    classNamePrefix='users-emails-input'
+                    placeholder={this.props.placeholder}
+                    components={this.components}
+                    getOptionValue={this.getOptionValue}
+                    formatOptionLabel={this.formatOptionLabel}
+                    defaultOptions={false}
+                    defaultMenuIsOpen={false}
+                    openMenuOnClick={false}
+                    loadingMessage={this.loadingMessage}
+                    onInputChange={this.handleInputChange}
+                    inputValue={this.props.inputValue}
+                    openMenuOnFocus={true}
+                    onFocus={this.onFocus}
+                    tabSelectsValue={true}
+                    value={values}
+                    aria-label={this.props.ariaLabel}
+                />
+                {this.props.showError && (
+                    <div className='InputErrorBox'>
+                        <FormattedMarkdownMessage
+                            id={this.props.errorMessageId}
+                            defaultMessage={this.props.errorMessageDefault}
+                            values={this.props.errorMessageValues || null}
+                            disableLinks={true}
+                        >
+                            {(message) => (
+                                <components.NoOptionsMessage>
+                                    {message}
+                                </components.NoOptionsMessage>
+                            )}
+                        </FormattedMarkdownMessage>
+                    </div>
+                )}
+            </div>
```

Comment:
Continued from above:
```suggestion
            </>
```